website/settings/defaults.py

```diff
@@ -1958,3 +1959,32 @@ class CeleryConfig:
 DS_METRICS_BASE_FOLDER = None
 REG_METRICS_OSF_TOKEN = None
 REG_METRICS_BASE_FOLDER = None
+
+
+@enum.unique
+class StorageLimits(enum.IntEnum):
+    OVER_CUSTOM = -1
+    DEFAULT = 0
+    APPROACHING_PRIVATE = 4.5 * 10 ** 9
+    OVER_PRIVATE = 5 * 10 ** 9
+
+    APPROACHING_PUBLIC = 45 * 10 ** 9
+    OVER_PUBLIC = 50 * 10 ** 9
+
+    @classmethod
+    def choices(cls):
+        return [(key.value, key.name) for key in cls if key.value >= 0]
+
+    @classmethod
+    def public_choices(cls):
+        return [cls.DEFAULT, cls.APPROACHING_PUBLIC, cls.OVER_PUBLIC]
+
+    @classmethod
+    def private_choices(cls):
+        return [cls.DEFAULT, cls.APPROACHING_PRIVATE, cls.OVER_PRIVATE]
+
+    @classmethod
+    def status(cls, storage_usage: int, public: bool):
+        choices = cls.public_choices() if public else cls.private_choices()
```

Comment:
If you are on a public node and you are `OVER` the `PRIVATE` storage limit but under the `APPROACHING_PUBLIC` limit, this should still return `OVER_PRIVATE`. This is because the front end and api need to know whether to prevent the node from being made private if it's over the limit. Similarly, if you're on a private node and you are `APPROACHING_PUBLIC`, it should report `APPROACHING_PUBLIC`. 

So it should return whatever state it's in regardless of the publicness of the node.